staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -500,6 +534,22 @@ func (e *Store) Update(ctx context.Context, name string, objInfo rest.UpdatedObj
 		doUnconditionalUpdate := newResourceVersion == 0 && e.UpdateStrategy.AllowUnconditionalUpdate()
 
 		if existingResourceVersion == 0 {
+			var cleanup func(context.Context, bool)
```

Comment:
nit: can I get a `createCleanup` here and an `updateCleanup` down below for future me?

Reply:
ACK.  In next push :)